packages/eslint-plugin-circuit-ui/renamed-package-scope/index.ts

```diff
@@ -66,21 +66,69 @@ export const renamedPackageScope = createRule({
       // See https://eslint.org/docs/latest/extend/selectors#known-issues
       const escapedFrom = from.replace('/', '\\u002F');
 
+      function fixPackageName(node: TSESTree.Node | TSESTree.Token) {
+        context.report({
+          node,
+          messageId: 'refactor',
+          data: { from, to },
+          fix(fixer) {
+            return fixer.replaceText(
+              node,
+              context.sourceCode.getText(node).replace(from, to),
+            );
+          },
+        });
+      }
+
       return Object.assign(visitors, {
         [`ImportDeclaration:has(Literal[value=/${escapedFrom}(?!-).*/])`]: (
           node: TSESTree.ImportDeclaration,
         ) => {
-          context.report({
-            node,
-            messageId: 'refactor',
-            data: { from, to },
-            fix(fixer) {
-              return fixer.replaceText(
-                node,
-                context.sourceCode.getText(node).replace(from, to),
-              );
-            },
-          });
+          const { source } = node;
+
+          if (source.type !== 'Literal') {
+            return;
+          }
+
+          fixPackageName(source);
+        },
+        [`ImportExpression:has(Literal[value=/${escapedFrom}(?!-).*/])`]: (
+          node: TSESTree.ImportExpression,
+        ) => {
+          const { source } = node;
+
+          if (source.type !== 'Literal') {
+            return;
+          }
+
+          fixPackageName(source);
+        },
+        [`CallExpression:has(Identifier[name=/(mock|requireActual|importActual)/]):has(Literal[value=/${escapedFrom}(?!-).*/])`]:
```

Comment:
@sirineJ I modified the matcher to target specific test-related functions instead to be able to support Vitest in addition to Jest.